src/logging/mod.rs

```diff
@@ -0,0 +1,104 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at https://mozilla.org/MPL/2.0/.
+
+use std::{io, ops::Deref};
+
+use failure::err_msg;
+use rocket::{Request, State};
+use slog::{self, Drain, Record, Serializer, KV};
+use slog_async;
+use slog_mozlog_json::MozLogJson;
+use slog_term;
+
+use app_errors::Result;
+use settings::*;
```

Comment:
Fwiw, I think I prefer importing specific types from each module explicitly, rather than everything in one go. It just makes it a bit more obvious to see where imports are coming from when you're reading the code.